style/bundle.css

```diff
@@ -10,8 +10,11 @@
 @import url(./components/overlay.css);
 
 @import url(./components/badge.css);
+@import url(./components/attachment.css);
+@import url(./components/relevant-badge.css);
```

Comment:
Wtf is this?? What does `relevant-badge` mean? What is irrelevant badge then?

Reply:
Dude, you could implement large size and icon-only (**if needed**) style for your existing badge🤦‍♂️